### ISSUE TITLE
[Component] [message] [Bug Report] 在本地 CDN 方式下,如何使用ElMessge 等组件

### ISSUE THREAD

**Issue by idoly** [element-plus/element-plus#8878]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.37`
- Element Plus Version: `2.2.9`
- Browser / OS: `Chrome/103.0.5060.114`
- Build Tool: `CDN`

### Related Component

- `el-message`

### Reproduction Link

[Link]()

### Steps to reproduce

```html
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <link rel="stylesheet" href="//unpkg.com/element-plus/dist/index.css" />

    <script src="https://unpkg.com/vue"></script>
    <script src="https://unpkg.com/axios"></script>
    <script src="https://unpkg.com/element-plus"></script>

    <title>demo</title>
</head>
<body>
  <div id="app">
    <el-button type="primary" @click="connect">connect</el-button>
  </div>
  <script>
    const { ElMessage } = ElementPlus

    createApp({
      setup() {
        const connect = () => {
          ElMessage.error("Warning, this is a warning message.");
        }

        return {
          connect
        }
      }
    }).use(ElementPlus).mount('#app')
  </script>
  </body>
</html>
```

### What is Expected?

当我点击connect按钮时,应弹出错误消息提示

### What is actually happening?

点击无反应,浏览器控制台输出:

vue:1796 Uncaught TypeError: Cannot read properties of null (reading 'appendChild')

element-plus:53125 Uncaught (in promise) ReferenceError: Cannot access 'instance' before initialization

### Additional comments

这段代码在JSFiddle和 CodePen测试,是没有问题的,但是本地用chrome或者edge,就不行

**Comment by idoly** [comment]
vue:1616 [Vue warn]: Unhandled error during execution of directive hook 
  at <BaseTransition onBeforeLeave=fn<onClose> onAfterLeave=fn appear=false  ... > 
  at <Transition name="el-message-fade" onBeforeLeave=fn<onClose> onAfterLeave=fn  ... > 
  at <ElMessage customClass="" center=false dangerouslyUseHTMLString=false  ... >

**Comment by daonatural** [comment]
将 `<script>` 标签 引入的cdn链接写在 body 元素加载之后即可

```

<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />

    <link rel="stylesheet" href="//unpkg.com/element-plus/dist/index.css" />
    <title>demo</title>
  </head>
  <body>
    <div id="app">
      <el-button type="primary" @click="connect">connect</el-button>
    </div>
    <script src="https://unpkg.com/vue"></script>
    <script src="https://unpkg.com/axios"></script>
    <script src="https://unpkg.com/element-plus"></script>
    <script>
      const { ElMessage } = ElementPlus;

      Vue.createApp({
        setup() {
          const connect = () => {
            ElMessage.error("Warning, this is a warning message.");
          };

          return {
            connect,
          };
        },
      })
        .use(ElementPlus)
        .mount("#app");
    </script>
  </body>
</html>
```

**Comment by btea** [comment]
或者这样使用
```ts
const connect = () => {
  ElMessage({
    type: 'error',
    message: 'Warning, this is a warning message.',
    appendTo: document.body
  });
}
```

**Comment by idoly** [comment]
好的好的 谢谢

**Comment by btea** [comment]
fixed #8730
